client/i18n/locales/english/intro.json

```
@@ -1613,7 +1613,9 @@
       },
       "learn-how-to-use-conditionals": {
         "title": "Learn How to Use Conditionals",
-        "intro": ["", ""]
+        "intro": [
+          "In this course, you will learn to use the Conditional sentences to describe hypothetical outcomes depending on the fact that certain conditions are met."
```

```suggestion
          "In this course, you will learn to use the conditional sentences to describe hypothetical outcomes depending on the fact that certain conditions are met."
```
I don't think it needs capitalising - it's not a proper noun.